### ISSUE TITLE
GEM: Set to beamscraper jaws broken in jaws manager

### ISSUE THREAD

**Issue by DominicOram** [ISISComputingGroup/IBEX#4731]
As a user on GEM I would like to be able to set my beamscraper jaws from my jaws manager. Currently this does not work. Make sure this is thoroughly tested so that it does not regress again.

**Comment by Tom-Willemsen** [comment]
consider hotfixing on GEM when done
